packages/flutter/lib/src/cupertino/page.dart

```diff
@@ -75,15 +76,22 @@ class CupertinoPageTransition extends AnimatedWidget {
     // TODO(ianh): tell the transform to be un-transformed for hit testing
     // but not while being controlled by a gesture.
     return new SlideTransition(
-      position: _outgoingPositionAnimation,
+      position: _secondaryPositionAnimation,
       child: new SlideTransition(
-        position: _incomingPositionAnimation,
-        child: new PhysicalModel(
-          shape: BoxShape.rectangle,
-          color: _kBackgroundColor,
-          elevation: 32,
+        position: _primaryPositionAnimation,
+        child: new DecoratedBox(
+          decoration: new BoxDecoration(
+            boxShadow: <BoxShadow>[
```

Comment:
(by which i mean, ContainerTransition would be to AnimatedContainer what FadeTransition is to AnimatedOpacity.)